pkgpanda/__init__.py

```diff
@@ -42,14 +41,27 @@
 version_regex = "^[a-zA-Z0-9@_+:.]+$"
 username_regex = "^dcos_[a-z0-9_]+$"
 
+dcos_sysctl_config = '/etc/sysctl.d/dcos.conf'
+
+
+def preserve_sysctl(sysctl_config_file, preserved_sysctl_map):
+    sysctl_setting = "\n".join(["{key} = {value}".format(key=key, value=value)
+                                for key, value in preserved_sysctl_map.items()])
+    write_string(sysctl_config_file, sysctl_setting)
+
+
+def activate_sysctl(sysctl_config_file):
+    # NOTE: '-e' flag ignores errors on invalid sysctl flag.
```

Comment:
Should probably not do the `-e` flag. If we do an invalid flag, we should probably hard error. We should catch that in CI, rather than having a typo in a sysctl ship in the product.